src/test/test-fd-util.c

```diff
@@ -348,7 +350,11 @@ static void test_close_all_fds(void) {
         }
         assert_se(r >= 0);
 
+#if HAVE_SECCOMP
         if (!is_seccomp_available()) {
```

Comment:
this should probably be refactored so that it can be called when building without seccomp

Reply:
It's a bit more churn as some types are not available when there is no <seccomp.h>, so the header needed some rework, but if you're fine with that sure -- I've just force-pushed a v2.